src/modules/vtol_att_control/standard.cpp

```diff
@@ -300,11 +296,9 @@ void Standard::update_transition_state()
 
 		}
 
-		// in fw mode we need the multirotor motors to stop spinning, in backtransition mode we let them spin up again
-		if (_flag_enable_mc_motors) {
-			set_max_mc(2000);
-			set_idle_mc();
-			_flag_enable_mc_motors = false;
+		// in back transition we need to start the MC motors again
+		if (_motor_state != ENABLED) {
+			_motor_state = set_motor_state(_motor_state, ENABLED);
```

Comment:
Thinking about passing the VT_FW_MOT_OFFID parameter as another argument here so that it become more clear that this methods sets the state of a motors selection.